packages/executor/src/services/UserOpValidation.ts

```diff
@@ -124,7 +124,7 @@ export class UserOpValidationService {
       this.provider
     );
     const errorResult = await entryPointContract.callStatic
-      .simulateValidation(userOp, { gasLimit: 10e6 })
+      .simulateHandleOp(userOp, AddressZero, BytesZero, { gasLimit: 10e6 })
```

Comment:
Pls move the gasLimit also to the constants file. We should make all of these configurable via the config.json. If no values are present, let the bundler use the default values. This will ensure we do not have to release a new version every time we need to change this.